lib/arclight/custom_document.rb

```diff
@@ -17,5 +17,17 @@ class CustomDocument < SolrEad::Document
       t.accessrestrict(path: 'archdesc/accessrestrict/p', index_as: %i[displayable])
       t.scopecontent(path: 'archdesc/scopecontent/p', index_as: %i[displayable])
     end
+
+    def to_solr(solr_doc = {})
+      super(solr_doc)
```

Comment:
This might be just a random ruby fact, but `super` calls the parent method with the arguments passed to the current method, so you can simply call `super` (without arguments) here and get the same result.